test/Sema/objc_attr_requires_module_3.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-build-swift -parse %s -Xfrontend -verify
+// REQUIRES: OS=linux-gnu
```

Comment:
> But, should I use `#if false` without any `REQUIRES`?

I think that's a good way to do it.  It would allow us to run the test on OS X, maximizing coverage.